src/WSEvents.cpp

```diff
@@ -1338,6 +1366,166 @@ void WSEvents::OnSourceFilterOrderChanged(void* param, calldata_t* data) {
 	self->broadcastUpdate("SourceFiltersReordered", fields);
 }
 
+/**
+ * A media source has started playing.
+ *
+ * Note: This event is only emitted when something actively controls the media/VLC source. In other words, the source will never emit this on its own naturally.
+ *
+ * @return {String} `sourceName` Source name
```

Comment:
There's a `@return` line missing for `sourceTypeId`, here and in several other places where `getMediaSourceData` is used.